qa/rpc-tests/dip4-coinbasemerkleroots.py

```diff
@@ -128,9 +229,55 @@ def test_getmnlistdiff_base(self, baseBlockHash, blockHash):
         assert_equal(d2["cbTx"], d.cbTx.serialize().hex())
         assert_equal(set([int(e, 16) for e in d2["deletedMNs"]]), set(d.deletedMNs))
         assert_equal(set([int(e["proRegTxHash"], 16) for e in d2["mnList"]]), set([e.proRegTxHash for e in d.mnList]))
+        assert_equal(set([QuorumId(e["llmqType"], int(e["quorumHash"], 16)) for e in d2["deletedQuorums"]]), set(d.deletedQuorums))
+        assert_equal(set([QuorumId(e["llmqType"], int(e["quorumHash"], 16)) for e in d2["newQuorums"]]), set([QuorumId(e.llmqType, e.quorumHash) for e in d.newQuorums]))
 
         return d
 
+    def test_dip8_quorum_merkle_root_activation(self, with_initial_quorum):
+        if with_initial_quorum:
+            self.nodes[0].spork("SPORK_17_QUORUM_DKG_ENABLED", 0)
+            self.wait_for_sporks_same()
+
+            # Mine one quorum before dip8 is activated
+            self.mine_quorum()
+
+        self.nodes[0].spork("SPORK_17_QUORUM_DKG_ENABLED", 4070908800)
+        self.wait_for_sporks_same()
+
+        cbtx = self.nodes[0].getblock(self.nodes[0].getbestblockhash(), 2)["tx"][0]
+        assert(cbtx["cbTx"]["version"] == 1)
+
+        assert(self.nodes[0].getblockchaininfo()["bip9_softforks"]["dip0008"]["status"] != "active")
+
+        while self.nodes[0].getblockchaininfo()["bip9_softforks"]["dip0008"]["status"] != "active":
+            self.nodes[0].generate(4)
```

Comment:
Make sure nodes are moving together and not lagging behind too much
```suggestion
            self.nodes[0].generate(4)
            self.sync_all()
```